streaming/src/protobuf/remote_call.proto

```diff
@@ -14,8 +14,8 @@ message ExecutionGraph {
     int32 parallelism = 2;
     NodeType node_type = 3;
     Language language = 4;
-    // serialized user function
-    bytes function = 5;
+    // serialized operator
+    bytes operator = 5;
```

Comment:
no need to build function in python worker?

Reply:
Put function build in `operator.load_operator`